README.rst

```diff
@@ -190,6 +190,23 @@ Available languages are stored in a mapping, and so best edited directly in Tuto
 
 Where the first key is the abbreviation of the language to use, "flag" is which flag icon is displayed in the user interface for choosing the language, and "name" is the displayed name for that language. The mapping above shows all of the current languages supported by Superset, but please note that different languages have different levels of completion and support at this time.
 
+Adding custom Row Level Security Filters to Superset
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to apply custom row level security filters to Superset, you can do so by using the
+setting SUPERSET_EXTRA_ROW_LEVEL_SECURITY_FILTERS in the config.yml file. This setting expects
+a list of dictionaries with the following structure:
+
+..  code-block:: yaml
+    
+    SUPERSET_EXTRA_ROW_LEVEL_SECURITY_FILTERS:
+        - schema: "xapi"
+          table_name: "video_plays"
+          role_name: "Open edX"
+          group_key": "xapi_course_id"
+          clause": 'xapi_course_id = "{}"'
+          filter_type": RowLevelSecurityFilterType.REGULAR
```

Comment:
This needs to be the string value of that enum, not the name of the enum itself:

```suggestion
          filter_type: "Regular" # or "Base"
```

Also think it's worth referencing some docs for this, but I didn't find anything good on Superset's site.. Found this instead? https://docs.preset.io/docs/row-level-security-rls